app/helpers/application_ui_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ApplicationUiHelper
+  # Create a edit button with icon
+  #
+  # @param path [String] The path of the link.
```

Comment:
If local one also need so flexible, I will not dare to write this helpers anymore, lol